app/Transformers/API/Chat/MessageTransformer.php

```diff
@@ -38,7 +38,8 @@ public function transform($message)
             'target_type' => $message->target_type,
             'target_id' => $message->target_id,
             'timestamp' => json_time($message->timestamp),
-            'content' => $message->content,
+            'content' => preg_replace('/\x01ACTION |[\x00-\x1f\x7F]/', '', $message->content),
+            'is_action' => preg_match('/\x01ACTION\s.*\x01/', $message->content),
```

Comment:
also in model. And using `starts_with` should be enough instead of full regexp (which also missing `^` and `$`).